lib/publiccloud/instance.pm

```diff
@@ -430,7 +430,8 @@ sub wait_for_ssh {
     # result display
     $sysout .= "\nTimeout $args{timeout} sec. expired" if ($duration >= $args{timeout});
     $instance_msg = "Check" . ($args{systemup_check} ? " SYSTEM " : " SSH ") . ($args{wait_stop} ? "DOWN" : "UP") .
-      ", $instance_msg, Duration: $duration sec.\nResult: $sshout . $sysout";
+      ", $instance_msg, Duration: $duration sec.\nResult: $sshout";
+    $instance_msg .= $sysout if defined($sysout);
```

Comment:
```suggestion
      ", $instance_msg, Duration: $duration sec.\nResult: $sshout" . $sysout;
```
this way, when sysout undefined, nothings more added.

Reply:
I don't understand your suggestion your suggested change just reverting what I have added to the old version when we will get perl error

Reply:
Sorry, not a full revertion: proposal was to put $sysout outside the double-quotes and concatenate the strings with the `.`, so when var is empty it is transparent. 
Probably the var. in the quotes caused issue when empty, but please go on with your changes.